blazor/diagram/connectors/segments/straight.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: Straight Segment in Blazor Diagram Component | Syncfusion
+description: Checkout and learn here all about Straight Segments in Syncfusion Blazor Diagram component and much more details.
```

Comment:
description: Check out and learn about Straight Segments in Syncfusion Blazor Diagram component and much more details.